### PR TITLE
[corert] Import System.Dianostics.Debug

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

File: mcs/class/corlib/corert/Debug.cs [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+			{
+				// FIXME should we g_error in that case?
+			}
 
-		public static void Fail (string message)
-		{
+			public void WriteCore(string message)
+			{
+				// FIXME replace by a call into the runtime, to g_critical for example
+				Console.Error.Write (message);
```
Comment (akoeplinger) [inline mcs/class/corlib/corert/Debug.cs]: I'm not sure we want stderr or g_critical here, wouldn't g_debug be the more equivalent?

**Comment by luhenry** [comment]
build

File: mcs/class/corlib/corert/Debug.cs [hunk 2]
```diff
@@ -1,17 +1,22 @@
-namespace System.Diagnostics.Private
+
+namespace System.Diagnostics
```
Comment (marek-safar) [inline mcs/class/corlib/corert/Debug.cs]: Unfortunately we cannot do that, you need to change the call site

Comment (luhenry) [inline mcs/class/corlib/corert/Debug.cs]: Yes, I was realising that while compiling `System.Core`... I will revert that

**Pull request merged**